Chutzpah/TestRunner.cs

```diff
@@ -305,10 +305,14 @@ private IChutzpahWebServerHost SetupWebServerHost(ConcurrentBag<TestContext> tes
             if (contextWithChosenServerConfiguration != null)
             {
                 var webServerConfiguration = contextWithChosenServerConfiguration.TestFileSettings.Server;

```

Comment:
The problem here now is that we are grabbing the webServerConfiguration from a context that may not have it enabled.

Reply:
Correct. When enabled is false it does not really do anything. See the comment below